### DISCUSSION TITLE
user-scoped custom parameters showing up in debugview but not in production

### DISCUSSION THREAD

**Discussion by hanzhenl** [invertase/react-native-firebase#6119] [Q&A]
Are there known causes for user-scope custom parameters (for example, user ID) to show up i debugview testing, but not in production reporting?

Another phenomenon I noticed is that only my own user ID which is used during testing appears in production reporting. Others users' custom parameters stay "not set".

Thank you.

**Comment by mikehardy** [comment]
This isn't an issue that has come up before in my memory of support here over a couple years now. Strange collection of behavior. What *has* come up is the debug view not being 100% reliable with regards to reporting custom analytics stuff, but I haven't heard of production-related logging irregularities other than timing maybe being delayed. None of that sounds exactly on target, but it's in the area.

Is it both platforms? If you can get a reproduction against the platform it's happening on (or pick one, if on both) you can get support from internal firebase team

https://github.com/firebase/quickstart-ios/tree/master/analytics
https://github.com/firebase/quickstart-android/tree/master/analytics

I'm guessing this is going to be a backend processing them of some sort, and this may be the quickest route to diagnosis + resolution
